test/test_data/mock_responses/scatter-plot.test-data.js

```diff
@@ -0,0 +1,819 @@
+/** @fileoverview Mock responses for scatter plot tests
```

Comment:
To me it seems a little more expensive to maintain than we ned to have a full-fledged mock response just to test different label configurations--now any time any part of the explore API changes, we have to update this file too.  I would move the existing MOCK_CLUSTER_RESPONSE to this file, and then make these responses just diffs of it.  I don't feel too strongly about this though, since it is nice having the MOCK_CLUSTER_RESPONSE in the same file as the tests...

```
MANY_LABELS_CLUSTER_RESPONSE = Object.assign(_cloneDeep(MOCK_CLUSTER_RESPONSE), {
   <<extra labels>>
})

Reply:
Testing maintainability is complex.  Diff'd mocks strike me as harder reason about in terms of fidelity to real data.  `MOCK_CLUSTER_RESPONSE` and `MANY_LABELS_MOCKS.CLUSTER_RESPONSE` are from two different studies, and using diff'd mocks instead of full mocks would require the maintainer to build more context and reason more about whether the diffs are really representative.

I'll keep an eye on these to assess where the two approaches help and hurt maintainability.